plaso/output/shared_elastic.py

```diff
@@ -360,6 +364,18 @@ def Close(self):
 
     self._client = None
 
+  def SetAdditionalFields(self, additional_fields):
+    """Set the additional field.
```

Comment:
style consistency nit: Set => Sets

Reply:
consistent with many of the other function name, changing them to match...